tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/LockFreeExponentiallyDecayingReservoir.java

```diff
@@ -54,10 +57,11 @@
 @Beta
 public final class LockFreeExponentiallyDecayingReservoir implements Reservoir {
 
+    private static final double SECONDS_PER_NANO = .000000001D;
```

Comment:
format nit to add separators:

```suggestion
    private static final double SECONDS_PER_NANO = .000_000_001D;
```